packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -81,15 +86,21 @@ async function ssrTransformScript(
   try {
     ast = await rollupParseAstAsync(code)
   } catch (err) {
-    if (!err.loc || !err.loc.line) throw err
-    const line = err.loc.line
-    throw new Error(
-      `Parse failure: ${
-        err.message
-      }\nAt file: ${url}\nContents of line ${line}: ${
-        code.split('\n')[line - 1]
-      }`,
-    )
+    // enhance known rollup errors
+    // https://github.com/rollup/rollup/blob/42e587e0e37bc0661aa39fe7ad6f1d7fd33f825c/src/utils/bufferToAst.ts#L17-L22
+    if (err.code === 'PARSE_ERROR') {
+      err.message = `Parse Failure: ${err.message}\n`
```

Comment:
```suggestion
      err.message = `Parse failure: ${err.message}\n`
```

Being very nitpicky here, but I like the lowercase "failure" before 😅

Reply:
Oh, didn't intend to change, but more like typo. Will update :smile: